dss/util/auth/fusillade.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import typing
+import requests
+
+from dss import Config
+from dss.error import DSSForbiddenException, DSSException
+from .authorize import Authorize
+
+logger = logging.getLogger(__name__)
+
+
+class Fusillade(Authorize):
+    def __init__(self):
+        self.session = requests.Session()
+        pass
+
+    def security_flow(self, authz_methods: typing.List[str], *args, **kwargs):
```

Comment:
As per other comment, the `authz_methods` parameter could use some explanation in the docstring